client/src/pages/admin/merge/MergePeople.tsx

```diff
@@ -497,12 +497,19 @@ const PersonColumn = ({
         <AdvancedSingleSelect
           fieldName="person"
           placeholder="Select a person to merge"
-          value={person}
+          value={
+            person
+              ? { ...person, name: Person.militaryName(person.name) }
+              : person
+          }
```

Comment:
Unnecessary change, please revert.

Reply:
If we want to maintain the previous behaviour where the last name is capitalized on the Merge People page, then I believe we do need this.
This line in specific covers edge cases when we come to this page with a name selected already

Reply:
See https://github.com/NCI-Agency/anet/pull/5006#discussion_r1871461728